gui/packages/ubuntupro/lib/pages/widgets/navigation_row.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class NavigationRow extends StatelessWidget {
+  const NavigationRow({
+    required this.onBack,
+    required this.onNext,
+    this.backText,
+    this.nextText,
+    super.key,
+  });
+
+  final void Function()? onBack;
+  final String? backText;
+  final void Function()? onNext;
+  final String? nextText;
+
+  @override
+  Widget build(BuildContext context) {
+    final lang = AppLocalizations.of(context);
+
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        OutlinedButton(
+          onPressed: onBack,
+          child: Text(backText ?? lang.buttonBack),
+        ),
+        FilledButton(
+          onPressed: onNext,
+          child: Text(nextText ?? lang.buttonNext),
+        ),
```

Comment:
I see both buttons as the same kind in the mockup for the Landscape skip page only (while all other pages have a filled button on the preferred action). Did you check with the designers on that? Because I see you're setting the button to be filled unconditionally.

Reply:
The other pages in the mockups use filled buttons for the primary action, so I figured we should do the same here as well. I'll ask about this.

Reply:
I trust you to check if the Design team and create a new PR if necessary to avoid blocking this one.